### ISSUE TITLE
darc should not know about release package layout

### ISSUE THREAD

**Issue by mmitche** [dotnet/arcade-services#2707]
The release package layout is generated during gather-drop. This package layout categorizes packages by their owner, which enables nuget pushes later on.

This is not ideal:
- Changes to package layout (where packages are built) require Maestro rollouts
- Maestro has no reason to know about the release process.

This release package layout should either be generated in staging, or ideally not exist at all in staging. It's yet another copy of the build outputs. It would be better for the release pipeline just to use the normal signed drop.

**Comment by tkapin** [comment]
What problem does this present to us? Is it affecting customers somehow?

**Comment by dkurepa** [comment]
From what I understood, `System.Drawing.Common` was not getting published since it was moved into the winforms repository, and darc wasn't aware of this

**Comment by mmitche** [comment]
Yeah, so that means that a change is required in Maestro for customer packages to be published. And it's unnecessarily complex.
